### PR TITLE
Added Auth0 to react-discovery-ui

### PR DESCRIPTION
**Pull request by abrock-accenture**

### CODE REVIEW

File: config/config.js [hunk 1]
```diff
@@ -4,3 +4,5 @@ window.BASE_URL = 'example.com'
 window.STREETS_TILE_LAYER_URL = 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png'
 window.LOGO_URL = 'https://placekitten.com/530/530'
 window.MAPBOX_ACCESS_TOKEN = ''
+window.AUTH0_DOMAIN = 'smartcolumbusos-demo.auth0.com'
+window.AUTH0_CLIENT_ID = 'sfe5fZzFXsv5gIRXz8V3zkR7iaZBMvL0'
```
Comment (ManApart) [inline config/config.js]: I assume this is a dummy client id?

Comment (abrock-accenture) [reply config/config.js]: This is the ID for our 'demo' instance of AUTH0.  We will be overriding it in a future card.

File: src/auth/react-auth0-wrapper.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+      if (isAuthenticated) {
+        const user = await auth0FromHook.getUser();
+        setUser(user);
+      }
+
+      setLoading(false);
+    };
+    initAuth0();
+    // eslint-disable-next-line
```
Comment (jeffgrunewald) [inline src/auth/react-auth0-wrapper.js]: is this comment needed?

Comment (abrock-accenture) [reply src/auth/react-auth0-wrapper.js]: That came in with the boilerplate provided by Auth0.  I'm sure we can remove it.

**Pull request merged**